integration_tests/tests/consistency/consistency_campaign_group_report.sql

```diff
@@ -8,7 +8,9 @@ with prod as (
         account_id,
```

Comment:
I just noticed all these consistency tests are grouped by the `account_id`. Wouldn't it be more impactful to test them on the grain of the report? For example, the campaign_group_report consistency test should be grouped by `campaign_group_id` instead of `account_id`. 

Would you be able to make this update in each of the consistency tests to ensure accurate use in future releases.

Reply:
I did a quick check and noticed all tests still pass, but it's more relevant to each of the individual reports instead of just testing on the account_id for each.

Reply:
updated and still passing! 

![image](https://github.com/user-attachments/assets/0c417eb1-1feb-47f0-b1e4-26e34a0bde3f)